.github/workflows/quality.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: '18.x'
+          node-version: '^18.x'
       - run: npm install
       - run: npm test
+      - run: npx eslint src/**
```

Comment:
I guess you can define it into the script,  I mean this src

Reply:
i defined inside of overrides:
"overrides": [
		{
		"files": [ "src/**/*.js" ],
	}
		],